code/modules/reagents/chemistry/holder.dm

```diff
@@ -733,11 +733,11 @@
 
 	return FALSE
 
-/datum/reagents/proc/has_reagent(reagent, amount = -1, needs_metabolizing = FALSE)
+/datum/reagents/proc/has_reagent(reagent, amount = -1, needs_metabolizing = FALSE, exact_type = TRUE)
```

Comment:
undocumented change

Reply:
not a game-affecting change. i don't think you need to document every single helper proc you add or change

Reply:
you don't